tka/sig.go

```diff
@@ -139,14 +152,17 @@ func (s *NodeKeySignature) Unserialize(data []byte) error {
 }
 
 // verifySignature checks that the NodeKeySignature is authentic, certified
-// by the given verificationKey, and authorizes the given nodeKey.
+// by the given verificationKey, and (for all signatures except SigCredential)
```

Comment:
Perhaps enumerate the ones that it's true for, rather than the exceptions, so it won't go stale the next time something is added 😄